sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -454,8 +455,9 @@ case class InputAdapter(child: SparkPlan) extends UnaryExecNode with CodegenSupp
       writer: Writer,
       verbose: Boolean,
       prefix: String = "",
-      addSuffix: Boolean = false): Unit = {
-    child.generateTreeString(depth, lastChildren, writer, verbose, prefix = "", addSuffix = false)
+      addSuffix: Boolean = false,
+      maxDepth: Int = TreeNode.maxTreeToStringDepth): Unit = {
+    child.generateTreeString(depth, lastChildren, writer, verbose, prefix, addSuffix, maxDepth)
```

Comment:
Prefix and addSuffix are ignored in the original code but I can't tell why.  Seems like they should be passed through from parent to child here.  I adjusted, but it might not be a bug or maybe should go in a separate PR
```suggestion
    child.generateTreeString(depth, lastChildren, writer, verbose, prefix = "", addSuffix = false, maxDepth)
```